### PR TITLE
(feat): enable chromium protocol for edge

### PR DESCRIPTION
**Pull request by lacell75**
## Proposed changes

Resolves features request #12385

## Types of changes

- [ ] Polish (an improvement for an existing feature)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (improvements to the project's docs)
- [ ] Specification changes (updates to WebDriver command specifications)
- [ ] Internal updates (everything related to internal scripts, governance documentation and CI files)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Backport Request

- [ ] This change is solely for `v9` and doesn't need to be back-ported
- [ ] Back-ported PR at `#XXXXX`

### CODE REVIEW

File: website/i18n/ar/docusaurus-plugin-content-docs/current/DriverBinaries.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If you are looking to set up mobile and other UI platforms, have a look into our
 
 ## Chromedriver
 
-To automate Chrome you can download Chromedriver directly on the [project website](http://chromedriver.chromium.org/downloads) or through the NPM package:
```
Comment (christian-bromann) [inline website/i18n/ar/docusaurus-plugin-content-docs/current/DriverBinaries.md:18]: I don't think this should be replaced with a page that leads to Edgedriver.

Comment (lacell75) [reply website/i18n/ar/docusaurus-plugin-content-docs/current/DriverBinaries.md]: my bad. Fixed

**Review by christian-bromann — APPROVED** [review]
Nice 👌

**Comment by christian-bromann** [comment]
@lacell75 do you think we can backport this to v8?

**Comment by lacell75** [comment]
> nevermind, I was able to cherry-pick the squash commit.

nice! I  just made one more adaptation

**Pull request merged**